src/__tests__/checkValidPriceRating.test.js

```diff
@@ -0,0 +1,12 @@
+import { checkValidPriceRating } from "../utils/checkValidPriceRating";
```

Comment:
The structure I'd have (which as it turns out makes for easier code review flows is something like:
```
- src
-- components
--- __tests__ (for component tests)
-- utils
--- __tests__ (for component tests)
```